### PR TITLE
docs(aio): added ngconf announcement

### PR DESCRIPTION
**Pull request by joeeames**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

announcement for ng-conf on aio

## Does this PR introduce a breaking change?
```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: aio/content/marketing/announcements.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
   {
-    "startDate": "2018-01-01",
-    "endDate": "2018-02-02",
-    "message": "Join us in Atlanta for ngATL<br/>Jan 30 - Feb 2, 2018",
-    "imageUrl": "generated/images/marketing/home/ng-atl.png",
-    "linkUrl": "http://ng-atl.org/"
+    "startDate": "2018-02-14",
+    "endDate": "2018-04-22",
+    "message": "Join us in for ng-conf<br/>Apr 18th-20th, 2018",
+    "imageUrl": "generated/images/marketing/home/ng_logo.png",
```
Comment (gkalpak) [inline aio/content/marketing/announcements.json]: Apparently the name should be `NG_logo.png`.

**Review by gkalpak — APPROVED** [review]

File: aio/content/marketing/announcements.json (hunk 1 above)
Comment (joeeames) [reply aio/content/marketing/announcements.json]: so should i create a new PR or update this one?

**Comment by aaronfrost** [comment]
@petebacondarwin Joe and I are not sure how to help this process. The preview works, though. 

Do you need anything else from Joe? Or can I help at all?

File: aio/content/marketing/announcements.json (hunk 1 above)
Comment (gkalpak) [reply aio/content/marketing/announcements.json]: This one :grin:
Looks good now, thx!

File: aio/content/marketing/announcements.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 [
   {
-    "startDate": "2018-01-01",
-    "endDate": "2018-02-02",
-    "message": "Join us in Atlanta for ngATL<br/>Jan 30 - Feb 2, 2018",
-    "imageUrl": "generated/images/marketing/home/ng-atl.png",
-    "linkUrl": "http://ng-atl.org/"
+    "startDate": "2018-02-14",
+    "endDate": "2018-04-22",
+    "message": "Join us in for ng-conf<br/>Apr 18th-20th, 2018",
```
Comment (gkalpak) [inline aio/content/marketing/announcements.json]: I just noticed the `in for` 😇
I assume it is a typo(?)

Comment (aaronfrost) [reply aio/content/marketing/announcements.json]: Yeah. It's a typo.

Comment (joeeames) [reply aio/content/marketing/announcements.json]: yeah. it is. "in" should be removed.

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
I fixed the typo, renamed the logo (to make it obvious that it is an ng-conf logo), squashed @joeeames' commits and added a small layout fix for certain screen-widths:

**Before:**
aio-ng-conf

**After:**
aio-ng-conf-wrap

The preview looks good (to me :grin:), @robwormald "verbally" LGTM'd the logo, so I am adding the `merge` label 😎

**Pull request closed without merging**